lib/tasks/db.rake

```diff
@@ -40,16 +43,52 @@ namespace :db do
     con.execute("ALTER ROLE #{aact_superuser} IN DATABASE #{aact_back_db} SET SEARCH_PATH TO ctgov, support, public, ctgov_beta;")
   end
 
+  task single_row_comparison: [:environment] do
+    # hardcoded table names with one row per study from excel file
+    file_table_names=['brief_summaries', 'calculated_values', 'categories',
+                      'designs', 'detailed_descriptions', 'eligibilities',
+                      'participant_flows', 'studies']
+
+    # finds table_names and their columns according to list in the file_table_names
+    sql = "SELECT t.table_name,
+                  c.column_name
+           FROM information_schema.tables AS t
+           INNER JOIN information_schema.columns AS c on c.table_name = t.table_name
+                                                  AND c.table_schema = t.table_schema
+           WHERE t.table_schema= 'ctgov' AND c.table_name IN (#{file_table_names.map { |e| "'#{e}'" }.join(', ')});"
+    all_columns = ActiveRecord::Base.connection.execute(sql).to_a
+
```

Comment:
exclude the blacklisted columns from all `all_columns` we should not include `id` `updated_at` `created_at`